### ISSUE TITLE
UAInAppMessagingDelegate Class Being Used Instead of Interface

### ISSUE THREAD

**Issue by benfrye** [urbanairship/xamarin-component#79]
`UAInAppMessaging` is expecting a subclass of `UAInAppMessageControllerDelegate` for it's MessageControllerDelegate rather than a class that implements the Interface `IUAInAppMessageControllerDelegate`.

**Comment by BrianBatchelder** [comment]
Where are you seeing this, @benfrye ? I see the following in UAInAppMessaging.h:
```
/**
 * A optional delegate for configuring and providing custom UI during message display.
 */
@property(nonatomic, weak, nullable) id<UAInAppMessageControllerDelegate> messageControllerDelegate;
```

**Comment by benfrye** [comment]
https://github.com/urbanairship/xamarin-component/blob/c6c36e7f45fd7e5ed68a3d19f93f7e861fed8c72/src/AirshipBindings.iOS/ApiDefinition.cs#L1590

**Comment by benfrye** [comment]
Shouldn't this be expecting an `IUAInAppMessageControllerDelegate`?

**Comment by BrianBatchelder** [comment]
@benfrye - we fixed this in 4.6.3.
